pkg/config/filter_test.go

```
@@ -23,128 +23,117 @@ func TestInvestigationEntryShouldRun(t *testing.T) { //nolint:maintidx // table-
+			name:   "empty filter (nil tree) passes",
+			filter: &InvestigationFilter{},
+			ctx:    baseCtx,
+			want:   true,
 		},
 		{
 			name: "nil context passes",
-			entry: &InvestigationEntry{
-				Name: "test",
-				When: &FilterNode{Field: FieldAlertName, Operator: OperatorIn, Values: []string{"chgm"}},
+			filter: &InvestigationFilter{
+				Filter: &FilterNode{Field: "AlertName", Operator: OperatorIn, Values: []string{"chgm"}},
```

golangci-lint (2.12.2)

[error] string `AlertName` has 12 occurrences, but such constant `FieldAlertName` already exists

(goconst)